app/assets/stylesheets/_overrides.scss

```diff
@@ -13,6 +13,11 @@ $container-max-widths: (
   xl: 1400px
 ) !default;
 
+#main-container {
+  min-height: calc(100vh - 300px);
```

Comment:
Could we subtract more than 300px? Right now you have to scroll to see the footer on bigger screens, but it should be completely visible initially.

Reply:
Yes! Fixed